src/Spout/Writer/Style/BorderBuilder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Box\Spout\Writer\Style;
+
+/**
+ * Class BorderBuilder
+ */
+class BorderBuilder
+{
+    /**
+     * @var Border
+     */
+    protected $border;
+
+    public function __construct()
+    {
+        $this->border = new Border();
+    }
+
+    /**
+     * @param string $style Border style @see BorderPart::allowedStyles
+     * @param string $color Border A RGB color code
+     * @param string $width Border width @see BorderPart::allowedWidths
+     * @return self
+     */
+    public function setBorderTop($style = Border::STYLE_SOLID, $color = Color::BLACK, $width = Border::WIDTH_THICK)
```

Comment:
I do not mind changing the order to the newly proposed ```color > width > style``` - the IDE should do that for me hopefully. So should we give this a try?